src/%IPM/DependencyAnalyzer/Problem.cls

```diff
@@ -52,10 +55,13 @@ Storage Default
 <Value>ReferencedModule</Value>
 </Value>
 <Value name="8">
-<Value>ReferencedClass</Value>
+<Value>ReferencedItem</Value>
 </Value>
 <Value name="9">
-<Value>ReferencedClassExists</Value>
+<Value>ReferencedFileType</Value>
+</Value>
+<Value name="10">
+<Value>ReferencedItemExists</Value>
```

Comment:
Why the order of properties is changed?
ReferencedClassExists was replaced by ReferencedItemExists
and place is changed from 9 to 10

Reply:
Hi Dmitry, good question! This is because we've re-named the ReferenceClass property to ReferenceItem property and replaced all references to that property. ReferenceItem is a better name since it includes not only classes but also include files, etc. Additionally, we added ReferenceFileType property, that's why the place is changed from 9 to 10

Reply:
Thanks Dmitry, this is a very solid point and I totally agree! However, this class, %IPM.DependencyAnalyzer.Problem, will only be used in running dependency analyzer, which will always create %New() instance of Problem item. That been said, I don't see a reason that is will corrupt existing data; even that's the case, those old data of Problem class will never be used for twice, since dependencyanalyzer always use new instance of Problem and heve %Save an old data.

Reply:
It's not about corrupt existing data, I understand, that it will not happen
it's about consistency, any changes in Storage should be correct, it does not matter if data is not used. 
Just best. practice